crates/bevy_winit/src/lib.rs

```diff
@@ -119,6 +119,21 @@ pub fn winit_runner(mut app: App) {
                         state: converters::convert_element_state(state),
                     });
                 }
+                WindowEvent::MouseWheel { delta, .. } => match delta {
+                    event::MouseScrollDelta::LineDelta(x, y) => {
```

Comment:
I think we want to distinguish between LineDelta and PixelDelta. They have different units and should probably be handled separately (or put behind an abstraction that accounts for that). Exposing both in a "raw" form does force devs to handle both cases, but It would probably be better to give them the option here.

Maybe
```rust
MouseWheel {
  unit: MouseScrollUnit,
  x: f32
  y: f32
} 
```

Reply:
I created an enum `MouseScrollUnit` and updated the `MouseWheel` struct to include it as you indicated above.

Forgot a comma so the checks failed, but I'm updating now...